src/smtml/expr.ml

```diff
@@ -440,6 +450,10 @@ let rec relop ty op hte1 hte2 =
     let base = Eval.binop (Ty_bitv 32) Add (Num (I32 base)) o in
     value (if Eval.relop ty op base n then True else False)
   | op, List l1, List l2 -> relop_list op l1 l2
+  | Gt, _, _ -> relop ty Lt hte2 hte1
+  | GtU, _, _ -> relop ty LtU hte2 hte1
+  | Ge, _, _ -> relop ty Le hte2 hte1
+  | GeU, _, _ -> relop ty LeU hte2 hte1
```

Comment:
This was already merged. But maybe, to avoid accidentally constructing `Gt|Ge` expressions we could remove this datatype from `Ty.Relop.t`? (It would perhaps best be addressed in a separate PR)